### DISCUSSION TITLE
User is empty

### DISCUSSION THREAD

**Discussion by goran-paunovic** [TheRealFlyingCoder/remix-auth-socials#10] [Q&A]
Followed your examples and the example I found on [devtech](https://devtools.tech/blog/add-social-authentication-to-remix-run-project-or-remix-server---rid---GEN4IbgWorJNeQL7Gkm8).
But when I call
```
const user = await authenticator.isAuthenticated(request, {
    failureRedirect: "/",
  });
```
user is always null.

**Comment by TheRealFlyingCoder** [comment]
Hey let me know if you figured this out, I assume you just weren't returning a user in your remix-auth implementation :)
